src/adapter/azure-functions/handler.ts

```diff
@@ -0,0 +1,59 @@
+import type {
+  Context as AzureFunctionsContext,
+  HttpRequest as AzureFunctionsHttpRequest,
+} from '@azure/functions'
```

Comment:
I am wondering about this dependency.
Is it ok to write @azure/functions in "dependency" in package.json?

Reply:
I don't understand this part either, when @yusukebe built the AWS Adaptor, the Forked Event Type was built in, so the Lambda@edge type was also forked.

Reply:
Yeah, we should avoid making it dependent on `@azure/functions`, even if it only provides types.